src/integer/z/properties.rs

```diff
@@ -219,3 +247,51 @@ mod test_inv {
         assert!(inv_zero.is_none());
     }
 }
+
+#[cfg(test)]
+mod test_is_zero {
+    use super::Z;
+    use std::str::FromStr;
+
+    /// ensure that is_zero returns `true` for `0`
+    #[test]
+    fn zero_detection() {
+        let zero = Z::from(0);
+
+        assert!(zero.is_zero());
+    }
+
+    /// ensure that is_zero returns `false` for non-zero values
+    #[test]
+    fn zero_rejection() {
+        let small = Z::from(2);
+        let large = Z::from_str(&format!("{}", (u128::MAX - 1) / 2 + 1)).unwrap();
```

Comment:
```suggestion
        let large = Z::from(u64::MAX);
```
suffices for large integer testing as its bigger than 2^62. Please remove this unnecessarily complex part also at all other parts in this PR.

Reply:
this is wanted, it is the least complex representation of 2^n with n>62 so that the last bits are all zero